src/openfisca/Data.js

```diff
@@ -195,259 +195,7 @@ const DATA = [
       ['2023-09-06', null],
       ['2024-09-06', null]
     ]
-  },
```

Comment:
Is everything below here duplicate data? why is it being deleted?

Reply:
@mischa-s It was made up data. I removed it to keep the tests clean.

Reply:
:+1: